tables/index.py

```diff
@@ -1130,8 +1145,14 @@ def cleanup_temp(self):
         # ...but the memory data cache is dirty now
         self.dirtycache = True
 
-    def get_neworder(self, neworder, src_disk, tmp_disk,
-                     lastrow, nslices, offset, dtype):
+    def get_neworder(self,
+                     neworder: np.ndarray,
+                     src_disk: EArray,
+                     tmp_disk: CArray,
```

Comment:
Maybe we could use more generic types for the two parameters above.
in principle anything supporting indexing should work.
I would suggest to use "Array"

Reply:
Thanks for the feedback. Changed both annotations to `Array`.

Reply:
@KoStehner there is some issue with the CI
I think that you need to use quotes: `Array` --> `"Array"`

Reply:
You are right, My bad. I fixed it